src/vm/class.h

```diff
@@ -1398,29 +1398,6 @@ class EEClass // DO NOT CREATE A NEW EEClass USING NEW!
     }
 
 public:
-
-    inline void SetDoesNotHaveSuppressUnmanagedCodeAccessAttr()
```

Comment:
These diffs are not part of your change - they are in master already. Could you please rebase them away?